libpod/container.go

```diff
@@ -297,6 +298,23 @@ func (c *Container) LogPath() string {
 	return c.logPath()
 }
 
+// IPAddress returns the IP addres of the container
```

Comment:
"addres of the container" to "address of the container".  Surprisingly it looks like lint is not happy with this.